content/2017/05/27/liveqa-biz.rst

```diff
@@ -0,0 +1,94 @@
+public: yes
+author: sondra
+type: event
+tags: [Event, Q&A, Tips, Slack, Startup, Business Development]
+image:
+  - src: 'blog/2017/liveqa-biz/desk-mobile.jpg'
+summary: |
+  Are you a startup founder
+  looking to develop a new web app
+  or a company interested in expanding your web services?
+  Here's your chance
+  to ask all those questions
+  that keep you up at night.
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: 'http://friends.oddbird.net'
+
+    Join the channel for our Q&A
```

Comment:
Are we concerned about the spacing between this button and the "Read More" button?

![screenshot 2017-05-26 18 18 54](https://cloud.githubusercontent.com/assets/552316/26514436/e1cafbf6-423f-11e7-9d52-55a5af6a24ef.jpg)